roles/check-php/defaults/main.yml

```diff
@@ -0,0 +1,180 @@
+---
+# [^ >=] is for system, but I found it only worked after the parens
+parens: '[ ]\{0,\}([^ >=]'
+php_syntax:
+  - '$[^S$D|\.|0-9|conf|user|_SESSION][A-Za-z]'
```

Comment:
Every Earth site is failing on this check, with results like this:

```
methane [[u'oP\x17$poj', u'N${G', u'$\x0es\x04\x1fEF\u05ca', u'oP\x17$pgj', u'\x1f3\u0421\x12$,t', u'5\x12cL$\x15e'], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], [], []]
```

Let me find where that is in the DB dump.

Reply:
Looks like it's in a cache table. So I think what we need to do is call `sql-dump` with `--structure-tables-key=common`. Will test that.

Reply:
c66b7a1 fixes that.